lib/project/results-model.coffee

```diff
@@ -210,10 +210,12 @@ class ResultsModel
       filePathInsertedIndex = binaryIndex(@paths, filePath, stringCompare)
       @paths.splice(filePathInsertedIndex, 0, filePath)
 
-    @matchCount += result.matches.length
-
-    @results[filePath] = result
-    @emitter.emit 'did-add-result', {filePath, result, filePathInsertedIndex}
+    bufferPromise = atom.project.buildBuffer(filePath)
+    bufferPromise.then (buffer) =>
+      result.lines = buffer.lines
```

Comment:
I'm not sure if I can tell `buildBuffer` just to build the lines I need. The API Docs are really missing a lot of information unfortunately: https://atom.io/docs/api/v1.6.2/Project.

However, from the modelside, I can just use the lines required :)